src/index.js

```diff
@@ -1 +1,11 @@
+import supportsProtoAssignment from './supportsProtoAssignment';
+
+// Display a developer friendly warning if __proto__ is not supported
+if (supportsProtoAssignment({}) === false) {
+  console.warn('This JavaScript environment does not support __proto__. ' +
+               'This means that react-proxy is unable to proxy React Components. ' +
+               'Features that rely on react-proxy, such as react-transform-hmr, ' +
+               'will not function as expected.');
```

Comment:
Getting back to these late :crying_cat_face: I definitely like that approach better. :+1: